src/ansys/openapi/common/_util.py

```diff
@@ -365,7 +365,7 @@ def handle_response(response: requests.Response) -> requests.Response:
 
 
 def generate_user_agent(package_name: str, package_version: str) -> str:
-    """Generate a user-agent string in the form <package info> <python info> <os info>.
+    """Generate a user-agent string in the form ``*<package info>*`` ``*<python info>*`` ``*<os info>*``.
```

Comment:
This has rendered as:

![image](https://user-images.githubusercontent.com/6451062/208759656-d3c6fd35-ab65-4867-aab4-dc64918ca906.png)

I don't know if it's possible to have emphasized literals in ReST, but if so, the asterisks might need to be outside?

Reply:
I've changed it to your proposed way to see if that works!

Reply:
![image](https://user-images.githubusercontent.com/6451062/209017626-f3c0549c-702b-47cb-bca1-a8da5cb2de63.png)

That hasn't worked either. I think it probably just needs to be either a double-backtick `literal`, or *emphasized*. I'll leave it to you which one is correct!

Reply:
@PipKat 
![image](https://user-images.githubusercontent.com/6451062/209159158-12b0c600-c313-4dd6-89ed-a9b1ff3b5d94.png)

I think this is probably ok?